test/integration/generated/additionalProperties/LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2021 Microsoft
+Copyright (c) 2022 Microsoft
```

Comment:
Why are these changes in this PR? Can we move it to another PR instead?

Reply:
I also have these changes on a PR I opened today, looks like the first PR of the year will have this when regenerating the tests swaggers.

Reply:
This is the result of regeneration.